modules/meta.analysis/R/approx.posterior.R

```diff
@@ -43,7 +43,9 @@ approx.posterior <- function(trait.mcmc, priors, trait.data=NULL, outdir=NULL, f
     
     dat    <- trait.mcmc[[trait]]
     vname  <- colnames(dat[[1]])
-    dat    <- as.matrix(dat)[,'beta.o']
+    if('beta.o' %in% vname) {
```

Comment:
I am not worried about the column name or the object class - just that we don't hack a fix for one case and forget another, resulting in a silent error (e.g. sampling from all parameters rather than the parameter of interest; here it seems we only subset beta.o if it exists).